kube-core/Cargo.toml

```diff
@@ -26,7 +26,7 @@ http = "0.2.2"
 [dependencies.k8s-openapi]
 version = "0.11.0"
 default-features = false
-features = []
+features = ["v1_20"]
```

Comment:
oh, this isn't ideal, we can't select this feature for users

Reply:
it's possible that the implementations of `Resource` and `ResourceExt` needs to be in `kube` to avoid having this feature issue here.

Reply:
Hmmm, that's unfortunate because that's the type that is causing issues for me. 

This only caused problems for running the tests, so maybe we can just specify the feature in CI?

Reply:
I've made this change if that works for you.

Reply:
Ah, yeah, that works.  The standard pattern we have is to select the pattern internally:

```toml
[dependencies.k8s-openapi]
version = "0.11.0"
default-features = false
features = []
[dev-dependencies.k8s-openapi]
version = "0.11.0"
default-features = false
features = ["v1_20"]
```